packages/client/src/index.tsx

```diff
@@ -19,6 +20,11 @@ if (!API_URL) {
 const httpService = new APIService(API_URL);
 const historyService = new HistoryService();
 
+// init Google Analytics
+if (process.env.NODE_ENV === 'production') {
+  ReactGA.initialize(process.env.GA_TRACKING_CODE || '');
```

Comment:
sorry to say this at the end, can you please lazy load __react-ga__ as will get split into another bundle and will be asynchronously loaded and won't block the app and if some one is not having tracking id then it will not be bundled at all

Reply:
No worries mate, made the change. Thanks

Reply:
awesome mate perfect, please squash and push and it should be good to merge!